scripts/build/build/targets.py

```diff
@@ -483,6 +483,7 @@ def AndroidTargets():
     yield target.Extend('arm64-chip-tool', board=AndroidBoard.ARM64, app=AndroidApp.CHIP_TOOL)
     yield target.Extend('x64-chip-tool', board=AndroidBoard.X64, app=AndroidApp.CHIP_TOOL)
     yield target.Extend('x86-chip-tool', board=AndroidBoard.X86, app=AndroidApp.CHIP_TOOL)
+    yield target.Extend('x86-matter-controller', board=AndroidBoard.X86, app=AndroidApp.MATTER_CONTROLLER)
```

Comment:
please generate both x86 and x64 targets, my machine is 64bits...

Reply:
acked